src/utils/getTeams.ts

```diff
@@ -6,11 +8,23 @@ export function getTeams(
   org: string
 ): string[] {
   const orgObj = GH_ORGS.get(org);
+  // TODO: include codecov
+  if (org === 'codecov') {
+    return [];
```

Comment:
Seems like this line is not hit by any tests?

Reply:
my test case appears to be wrong, fixing it now

Reply:
fixed now.